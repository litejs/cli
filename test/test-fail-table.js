
it("should fail on invalid table", [
	[1],
	[2,3]
], function(assert) {
	assert.end()
})

