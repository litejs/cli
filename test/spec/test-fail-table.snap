Error: Command failed: node --no-extra-info-on-fatal-exception -r ./test.js test/test-fail-table.js

test.js:247
				if (item.f.length > i.r.length + 2 || i.r.length !== spliceData[2].r.length) throw "Invalid data for: " + i.n
				                                                                             ^
Invalid data for: should fail on invalid table
(Use `node --trace-uncaught ...` to show where the exception was thrown)
